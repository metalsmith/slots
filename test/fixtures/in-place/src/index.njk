---
title: Nothing else matters
---
<h1>{{ title }}</h1>

{{ slots.footer.contents | safe }}

---
slot: footer
---
<footer>
footer-content
</footer>
