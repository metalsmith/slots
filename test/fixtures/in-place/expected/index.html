<h1>Nothing else matters</h1>

<footer>
footer-content
</footer>